subsys/disk/disk_access_spi_sdhc.c

```diff
@@ -748,6 +748,87 @@ static int sdhc_spi_write(struct sdhc_spi_data *data,
 	return err;
 }
 
+/* this function is optimized to write multiple blocks */
+static int sdhc_spi_write_multi(struct sdhc_spi_data *data,
+	const uint8_t *buf, uint32_t sector, uint32_t count)
+{
+	int err;
+	uint32_t addr;
+	uint8_t block[SDHC_CRC16_SIZE];
+
+	err = sdhc_map_disk_status(data->status);
+	if (err != 0) {
+		return err;
+	}
+
+	sdhc_spi_set_cs(data, 0);
```

Comment:
This way of handling SPI SDHC CS was removed in commit c64e0ce923cde54415d203e447a1dc62894ed2b0.

Reply:
replaced with 	`spi_release(data->spi, &data->cfg);`

Reply:
No, `spi_release(data->spi, &data->cfg);` should only be used where the `disk_access_spi_sdhc.c` used to de-assert the CS line. I believe this call can be removed all together.